streamer/src/nonblocking/quic.rs

```diff
@@ -54,6 +58,30 @@ const CONNECTION_CLOSE_REASON_EXCEED_MAX_STREAM_COUNT: &[u8] = b"exceed_max_stre
 const CONNECTION_CLOSE_CODE_TOO_MANY: u32 = 4;
 const CONNECTION_CLOSE_REASON_TOO_MANY: &[u8] = b"too_many";
 
+// A sequence of bytes that is part of a packet
+// along with where in the packet it is
+struct PacketChunk {
+    pub bytes: Bytes,
+    // The offset of these bytes in the Quic stream
+    // and thus the beginning offset in the slice of the
+    // Packet data array into which the bytes will be copied
+    pub offset: usize,
+    // The end offset of these bytes in the Quic stream
+    // and thus the end of the slice in the Packet data array
+    // into which the bytes will be copied
+    pub end_of_chunk: usize,
```

Comment:
Is `end_of_chunk` = `bytes.len() + offset`?

Reply:
The max of that over all chunks in the packet, yup. Same logic as the old code.